courses/free-courses-it.md

```diff
@@ -123,6 +124,7 @@
 
 ### Java
 
+* [Algoritmi e Strutture Dati Java](http://www.vitoantoniobevilacqua.it/wiki/index.php?title=Pagina_principale#Dispense_di_Algoritmi_e_Strutture_Dati_in_Java) - V. Bevilacqua (Politecnico di Bari)  
```

Comment:
:question: I'm not sure if is nice have a link to an anchor wiki or maybe use without it and move to other category

Reply:
The wiki contains tons of material about computer science since it is about the Bioengineering Course of the Politecnico di Bari (my university). Sadly for how it was structured that specific section I linked , it contains an entire course about Java and Data Structures, but it is not possible to isolate it in any way from the rest of the material (which anyway is always related to CS).
If you consider this the wrong place, please let me know where I can move this reference. For me it was the right place, since for italian speaker it is comprehensible what the section is about and what are the contents

Reply:
thanks for the explanation! We'll accept it, but you'll understand why we need to check.